app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItemComponents.kt

```diff
@@ -240,6 +242,9 @@ internal fun MessageDecryptionFailure(
                 text = stringResource(R.string.label_learn_more)
             )
             VerticalSpace.x4()
+
+            if (conversationProtocol !is Conversation.ProtocolInfo.Proteus) return@Column
```

Comment:
in fact this is the only changes in Compose, all others - just refactoring to make it more readable